src/assets/sass/variables/colors.scss

```
@@ -75,3 +75,6 @@ $white: #fff;
 
 $dropdown-text: #1d192b;
 $dropdown-hover: #c1cbdc;
+
+$select-disabled-bg: #f5f5f5;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)